test/eventbridge_rule_with_sns_fanout/config/resources.yaml

```diff
@@ -0,0 +1,2 @@
+topic_name: "aws-weathertop-central-sns-fanout-topic"
```

Comment:
Please add a one-line reminder of how the topic and bucket will be used, and links to the spec/docs that describe them.